### PR TITLE
[WIP] Improve cluster doc

### PR DESCRIPTION
**Pull request by nqb**
# Description
Add missing information from the cluster doc.

# Impacts
Doc.

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 1]
```diff
@@ -1117,6 +1117,23 @@ Then reload the file and reboot the server:
...
   # reboot
 
+
+IP addresses in a cluster environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DHCP and DNS services
+^^^^^^^^^^^^^^^^^^^^^
+
+In registration and isolation networks, DHCP and DNS servers are physical IP addresses of *all* cluster members.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: I think this should be rephrased - it's not clear to me.

Perhaps:

In registration and isolation networks, the DHCP and DNS services are bound to the physical IP address of each cluster member.

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 2]
```diff
@@ -1117,6 +1117,23 @@ Then reload the file and reboot the server:
...
+
+
+Clients SNMP
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: SNMP Clients

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 3]
```diff
@@ -1117,6 +1117,23 @@ Then reload the file and reboot the server:
...
+^^^^^^^^^^^^
+
+If you use SNMP in a cluster environment, you will need to allow physical IP addresses of **all** cluster members to query your network devices.
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: your network devices (switches, WiFi controllers, etc.).

**Comment by nqb** [comment]
TODO: mentioned active_active.dns_on_vip_only

**Comment by nqb** [comment]
@extrafu: done

**Pull request merged**